types/glob/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "@types/minimatch": "^5.1.2"
+        "minimatch": "^6.1.6"
```

Comment:
this can't happen IMO, unless there is a specific reason. Native module uses 5.* version:
https://github.com/isaacs/node-glob/blob/main/package.json#L23